### PR TITLE
Restore FEM to be static instance

### PR DESCRIPTION
**Pull request by riverlijunjie**
### Details:
  - Restore Frontend Manager to static instance from dynamic instance

### Tickets:
 - 87912

### CODE REVIEW

File: src/frontends/common/src/manager.cpp [hunk 1]
```diff
@@ -245,36 +245,11 @@ FrontEnd::Ptr FrontEndManager::load_by_model(const std::vector<ov::Any>& variant
...
-}
-
-}  // namespace
-
 FrontEndManager::Ptr get_frontend_manager() {
-    return get_frontend_manager_holder().get();
+    static std::shared_ptr<FrontEndManager> manager = nullptr;
+    if (!manager)
+        manager = std::make_shared<FrontEndManager>();
+    return manager;
```
Comment (ilya-lavrenov) [inline src/frontends/common/src/manager.cpp:252]: looks like we have not had this function before as well as changes in .hpp file https://github.com/openvinotoolkit/openvino/pull/11650/files#diff-9c901a405d3b2bb03116cbbdf5b52f607139c4c0690fafb81f96355fc02bce9e

Comment (riverlijunjie) [reply src/frontends/common/src/manager.cpp:252]: Original version puts this static Frontend Manager instance in ie_network_reader.cpp, which is a local function. Why we not put it into a global interface to benefit more potential requirement?

Comment (riverlijunjie) [reply src/frontends/common/src/manager.cpp:252]: ok, no problem.

Comment (riverlijunjie) [reply src/frontends/common/src/manager.cpp]: done

**Review by ilya-lavrenov — APPROVED** [review]

**Review by tomdol — APPROVED** [review]

**Review by praasz — APPROVED** [review]

**Comment by riverlijunjie** [comment]
> There is no any description of why we are doing this. Is it a quick fix solution for some well-known bug? Are we going to provide another solution that makes FEM unloadable in the future?

Dynamic Front End Management (FEM) is very risky for unknow use cases beyond the scope of CI validation. And dynamic FEM cannot totally solve OV library unloading issue. Team agree to revert both master branch and 2022.2 release branch to static FEM first and continue investigating OV library unloading issue.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**